api/app_analytics/split_testing/views.py

```diff
@@ -0,0 +1,96 @@
+from django.db.models import F, QuerySet
+from django.utils.decorators import method_decorator
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import mixins, viewsets
+from rest_framework.generics import ListAPIView
+
+from environments.authentication import EnvironmentKeyAuthentication
+from environments.permissions.permissions import EnvironmentKeyPermissions
+from features.models import FeatureState
+
+from .models import ConversionEventType, SplitTest
+from .permissions import ConversionEventTypePermissions, SplitTestPermissions
+from .serializers import (
+    ConversionEventSerializer,
+    ConversionEventTypeQuerySerializer,
+    ConversionEventTypeSerializer,
+    SplitTestQuerySerializer,
+    SplitTestSerializer,
+)
+
+
+class ConversionEventViewSet(viewsets.GenericViewSet, mixins.CreateModelMixin):
```

Comment:
Sorry, I'd rather this was a `CreateAPIView` - I'm not too sure why really, it's more of a feeling. I tend to think the `ViewSet` classes should be reserved for dashboard crud management, whereas this is used by the SDK.

Reply:
No problem.